frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template.tsx

```diff
@@ -87,12 +89,25 @@ const VMTemplateTableHeader = () =>
       {
         title: '',
       },
+      {
+        title: '',
+      },
     ],
     tableColumnClasses,
   );
 
 VMTemplateTableHeader.displayName = 'VMTemplateTableHeader';
 
+const getCreateLink = (
```

Comment:
can you please make a reusable funtion which includes also import and basic vm. 

Basically this funtion + createLink from vm.tsx + enums for all the stuff

Reply:
it could live probably under constants?